docs/src/project.md

```diff
@@ -137,3 +137,17 @@ pkg> activate .
 pkg> add Package1 Package2 ...
 ```
 Julia will automatically make the Project.toml and Manifest.toml files for you as you add packages used by your project.
+
+## Suppressing Update Messages
```

Comment:
Yeah, I am also not sure where to put this information. In the docs, especially here, it may be hard to discover. I was thinking to put this in the same bubble we have at the very start of `index.md` that shows the latest new features. But it is too long to put it there maybe...?

An alternative is to have this kind of information at the end of the update message itself (and I'll make sure I always retain this information there). However, let's make it as concise as possible. Let's say something as short as:
"pro hint: set the evironment variable `DRWATSON_UPDATE_MSG=0` (prior to `using DrWatson`) to turn off these update messages!"

I am not sure, let me know what you think.

Reply:
How about putting it somewhere in the docs with a linkable heading and then putting something in the update message like "To disable future update messages see http://..."

Reply:
Seems fitting for a `Misc` or `FAQ` section if you had one.

Reply:
A FAQ section was a nice idea, but the way you did it now is also nice for me, having a section aobut install and update :) thanks a lot!